py-polars/polars/expr/datetime.py

```diff
@@ -1493,9 +1507,20 @@ def days(self) -> Expr:
         """
         return wrap_expr(self._pyexpr.duration_days())
 
+    @deprecate_renamed_function("total_hours", version="0.19.3")
```

Comment:
The versions should be updated here and everywhere else.

```suggestion
    @deprecate_renamed_function("total_hours", version="0.19.13")
```